egauge-rb.gemspec

```diff
@@ -10,18 +10,9 @@ Gem::Specification.new do |spec|
 
   spec.summary       = %q{Ruby client for egauge inverter}
   spec.description   = %q{Ruby client for egauge inverter}
-  spec.homepage      = "http://eguage.com/maybe"
+  spec.homepage      = "https://github.com/ArcadiaPower/egauge-rb"
```

Comment:
LOL

Reply:
Yeah there's going to be some funny artifacts from when I was building this thing.